src-ui/package.json

```diff
@@ -78,5 +78,6 @@
       "lmdb",
       "msgpackr-extract"
     ]
-  }
+  },
+  "packageManager": "pnpm@10.15.0+sha512.486ebc259d3e999a4e8691ce03b5cac4a71cbeca39372a9b762cb500cfdf0873e2cb16abe3d951b1ee2cf012503f027b98b6584e4df22524e0c7450d9ec7aa7b"
 }
```

Comment:
Please dont include unrelated changes

Reply:
Every time i run the init task the package manager ist requesting one.
Do you want this on a seperate PR?